Scripts/WikiUpdater.cs

```diff
@@ -0,0 +1,415 @@
+        var heading = sectionName != null ? $"WIKI_HEADING_{sectionName}" : null;
+        var body = sectionName != null ? $"WIKI_{pageName}_{sectionName}" : $"WIKI_{pageName}_INTRO";
+
+        return new Wiki.Page.Section(heading, body);
+    }
+
+    /// <summary>
+    ///   Converts HTML for a single paragraph into BBCode. Paragraph must not contain lists, headings, etc.
+    /// </summary>
+    private static string ConvertTextToBbcode(string paragraph)
+    {
+        // Process our custom BBCode first
+        foreach (var compound in CustomBbcodeCompounds)
+        {
+            var compoundText = compound.ToLowerInvariant().Replace(" ", string.Empty);
+            paragraph = paragraph.Replace(
+                $"<b>{compound}</b>",
+                $"[thrive:compound type=\"{compoundText}\"][/thrive:compound]");
+        }
+
+        return paragraph
+            .Replace("\n", string.Empty)
+            .Replace("<b>", "[b]")
+            .Replace("</b>", "[/b]")
+            .Replace("<i>", "[i]")
+            .Replace("</i>", "[/i]")
+            .Replace("\"", "\\\"");
+    }
+
+    /// <summary>
+    ///   Inserts into en.po the English translations for all the translation keys in a list of wiki pages.
+    /// </summary>
+    private static async Task InsertTranslatedPageContent(List<TranslationPair> pages,
+        CancellationToken cancellationToken)
+    {
+        // Create the whole list of values to replace first, then replace asynchronously based on read lines
+        var translationPairs = new Dictionary<string, string>();
+
+        foreach (var page in pages)
+        {
+            var untranslatedPage = page.UntranslatedPage;
+            var translatedPage = page.TranslatedPage;
+
+            // Translate page names
+            translationPairs.Add(untranslatedPage.Name, translatedPage.Name);
+
+            for (var j = 0; j < untranslatedPage.Sections.Count; j++)
+            {
+                var untranslatedSection = untranslatedPage.Sections[j];
+                var translatedSection = translatedPage.Sections[j];
+
+                // Translate body sections
+                translationPairs.Add(untranslatedSection.SectionBody, translatedSection.SectionBody);
+
+                if (untranslatedSection.SectionHeading != null && translatedSection.SectionHeading != null)
+                {
+                    // Translate headings if present and not already in the list to translate
+                    translationPairs.TryAdd(untranslatedSection.SectionHeading, translatedSection.SectionHeading);
+                }
+            }
+        }
+
+        var reader = File.ReadLinesAsync(ENGLISH_TRANSLATION_FILE, Encoding.UTF8, cancellationToken);
+        var writer = new StreamWriter(File.Create(TEMP_TRANSLATION_FILE), new UTF8Encoding(false));
+
+        var isReplacingValue = false;
+        var isFuzzyValue = false;
+        await foreach (var line in reader)
+        {
+            if (line == string.Empty)
```

Comment:
```suggestion
            if (string.IsNullOrEMpty(line))
```

Should be used instead of compare against empty as that's the way it is done in the Thrive codebase.